Dockerfile

```diff
@@ -82,8 +82,8 @@ RUN mkdir -p ${APP_ROOT} && \
     curl -L https://github.com/ManageIQ/manageiq-ui-self_service/tarball/${REF} | tar vxz -C ${SSUI_ROOT} --strip 1
 
 ## Add ManageIQ source from local directory (dockerfile development) or from Github (official build)
-# ADD . ${APP_ROOT}
-RUN curl -L https://github.com/ManageIQ/manageiq/tarball/${REF} | tar vxz -C ${APP_ROOT} --strip 1
+ADD . ${APP_ROOT}
```

Comment:
@chriskacerguis
I know these discussions about COPY vs ADD  for a long time (BTW COPY was once declared as deprecated, but it stayed), so I checked this claim more thoroughly this time.
 
your comment above is a bit misleading,
True in some best practices blogs you can find preference to use COPY over ADD, but this is mainly due to the "magical"  part in the ADD command (around URLs and tar files) , and the general approach is "when you use something understand what it is doing".

About the image size - this comment usually refer to the case of adding a big tar file (which adds a layer to the docker image), and in a separate command deleting it (it will remain in one of the docker image snapshots ... there for will increase overall image size).
This is not the case here.

I even tested myself by constructing a simple Dockerfile that only adds the manageiq repo to an image , once using ADD and once using COPY and the image size is exactly the same.

Bottom line I don't really care in this use case what to be used (although personally I prefer ADD), but if we change it than it should be for the right reason.